randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -180,6 +180,8 @@ Extra - asset_id: collision_camera_003
       Any of the following:
           Flash Shift or Gravity Suit or Spider Magnet or Simple IBJ or Use Spin Boost
           Grapple Beam and Grapple Movement (Beginner)
+          # Damage Boost (Beginner): https://youtu.be/qIanrM7cc3I
+          Damage Boost (Beginner)
```

Comment:
Should this have a slide jump on it?

Reply:
I'm starting up a file to test it right now.

Reply:
Just confirmed it, no need to slide.

Reply:
I'm not making it with a jump, but you can dash melee it.